.circleci/config.yml

```diff
@@ -40,6 +40,18 @@ executors:
     docker:
       - image: grafana/wix-toolset-ci:v3
 
+commands:
+  install-google-cloud-sdk:
+    description: "Install Google Cloud SDK"
+    steps:
+      - run:
+          command: |
```

Comment:
Should have a name defined. Otherwise the name will be the command, see https://app.circleci.com/pipelines/github/grafana/grafana/4713/workflows/6c3004fd-30f3-466a-8610-c9fe2459a3f2/jobs/320880